iterations.r

```diff
@@ -52,18 +52,39 @@ source("gql-queries/vol_qry.r")
 
 stations_metadata_df %>% 
   filter(latestData > Sys.Date() - days(7)) %>% 
-  sample_n(1) %$% 
+  sample_n(1) %$% # sample a random station
   vol_qry(
     id = id,
     from = to_iso8601(latestData, -4),
     to = to_iso8601(latestData, 0)
   ) %>% 
   GQL(., .url = configs$vegvesen_url) %>%
   transform_volumes() %>% 
-  ggplot(aes(x=from, y=volume)) + 
+  ggplot(aes(x=from, y=volume, group=1)) + 
   geom_line() + 
-  theme_classic()
-
+  theme_classic() 
 
+### 6: Adjust codes so that we include the name of the selected traffic station
 
+# Sample a random station from the last 7 days
+station_sample <- stations_metadata_df %>% 
+  filter(latestData > Sys.Date() - days(7)) %>% 
+  sample_n(1)
+
+# Query and retrieve traffic data for the selected station
+traffic_data <- vol_qry(
+  id = station_sample$id,
+  from = to_iso8601(station_sample$latestData, -4),
+  to = to_iso8601(station_sample$latestData, 0)
+) %>% 
+  GQL(.url = configs$vegvesen_url) %>%
+  transform_volumes()
```

Comment:
Add this to the pipe:
```
mutate(from = as.POSIXct(from, format="%Y-%m-%dT%H:%M:%S", tz="CET"))
```

This will enable you to easily make the x-axis cleaner later on.